bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/action/SendMailActions.java

```diff
@@ -110,20 +110,30 @@ public class SendMailActions implements ThingActions {
 
     public static boolean sendMail(ThingActions actions, @Nullable String recipient, @Nullable String subject,
             @Nullable String text) {
-        return SendMailActions.sendMail(actions, recipient, subject, text, List.of());
+        return SendMailActions.sendMailWithAttachments(actions, recipient, subject, text, List.of());
     }
 
     public static boolean sendMail(ThingActions actions, @Nullable String recipient, @Nullable String subject,
             @Nullable String text, @Nullable String urlString) {
+        return SendMailActions.sendMailWithAttachment(actions, recipient, subject, text, urlString);
+    }
+
+    public static boolean sendMailWithAttachment(ThingActions actions, @Nullable String recipient,
+            @Nullable String subject, @Nullable String text, @Nullable String urlString) {
         List<String> urlList = new ArrayList<>();
         if (urlString != null) {
             urlList.add(urlString);
         }
-        return SendMailActions.sendMail(actions, recipient, subject, text, urlList);
+        return SendMailActions.sendMailWithAttachments(actions, recipient, subject, text, urlList);
     }
 
     public static boolean sendMail(ThingActions actions, @Nullable String recipient, @Nullable String subject,
             @Nullable String text, @Nullable List<String> urlStringList) {
+        return SendMailActions.sendMailWithAttachments(actions, recipient, subject, text, urlStringList);
+    }
+
+    public static boolean sendMailWithAttachments(ThingActions actions, @Nullable String recipient,
+            @Nullable String subject, @Nullable String text, @Nullable List<String> urlStringList) {
         return ((SendMailActions) actions).sendMailWithAttachments(recipient, subject, text, urlStringList);
     }
 
```

Comment:
In line 144 below: Parameter name is `html`, but it's documented as `htmlContent`. Should it be fixed also? See https://next.openhab.org/addons/bindings/mail/#rule-action

Reply:
In line 152 below: Parameter name is `url`, but it's documented as `URL`. Should it be fixed also?

Reply:
The same with `urlList` - for all overloads.

Reply:
Changed for `htmlContent`.

Reply:
For `URL`, I will rather update the documentation.

Reply:
Done